tests/parametric/test_span_sampling.py

```diff
@@ -753,3 +753,78 @@ def test_entire_trace_dropped_when_dropping_policy_is_active018(self, test_agent
         traces = test_agent.wait_for_num_traces(0, clear=True)
 
         assert len(traces) == 0
+
+
+    @bug(context.library == "cpp", reason="APMAPI-1545")
+    @bug(context.library == "golang", reason="APMAPI-1545")
+    @bug(context.library == "nodejs", reason="APMAPI-1545")
+    @bug(context.library == "php", reason="APMAPI-1545")
+    @bug(context.library == "ruby", reason="APMAPI-1545")
+    @bug(context.library == "python", reason="APMAPI-1545")
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_TRACE_PROPAGATION_STYLE": "datadog",
+                "DD_SPAN_SAMPLING_RULES": json.dumps([{"name": "web.request", "sample_rate": 1.0}]),
+                "DD_TRACE_SAMPLING_RULES": json.dumps([{"name": "web.request", "sample_rate": 1.0}]),
+            }
+        ],
+    )
+    def test_single_rule_with_head_and_rule_trace_sampling_019(self, test_agent, test_library):
```

Comment:
All tests seem to finish with a number, not sure why, so I did so too :noideawhatiamdoing: